test/client/form-cache.spec.js

```diff
@@ -86,40 +194,179 @@ describe( 'Client Form Cache', () => {
 
     } );
 
-    /*
-    describe( 'in cached state', function() {
-        
-        it( 'initializes succesfully', function( done ) {
+    describe( 'last-saved records', () => {
```

Comment:
The diff algorithm really botched this. I highly recommend just looking at the before/after if there’s confusion about what changed.